libs/server/Metrics/Latency/GarnetLatencyMetricsSession.cs

```diff
@@ -24,6 +24,16 @@ public GarnetLatencyMetricsSession(GarnetServerMonitor monitor)
             Init();
         }
 
+        public void Return()
+        {
+            foreach (var cmd in defaultLatencyTypes)
+            {
```

Comment:
Better to have a Return method within LatencyMetricsEntrySession that this method can invoke to maintain the abstraction? That way, invoking Return will become obvious to any future users of LatencyMetricsEntrySession.

Reply:
Done